source/MaterialXRenderHw/WindowCocoaWrappers.h

```diff
@@ -19,6 +19,7 @@ void NSUtilShowWindow(void* pWindow);
 void NSUtilHideWindow(void* pWindow);
 void NSUtilSetFocus(void* pWindow);
 void NSUtilDisposeWindow(void* pWindow);
+float NSUtilGetScaleFactor(void* pWindow);
```

Comment:
Since this didn't end up being part of the fix, I'd recommend omitting this for clarity, and we can add such a function when it's needed in the future.